src/tokens/TokenBalances.tsx

```diff
@@ -310,6 +312,7 @@ function TokenBalancesScreen({ navigation, route }: Props) {
       <AnimatedSectionList
         contentContainerStyle={{
           paddingBottom: insets.bottom,
+          opacity: listHeaderHeight > 0 ? 1 : 0,
```

Comment:
snuck this one in - this prevents the items being pushed down after the list header is measured